packages/eslint-plugin-shopify/lib/config/rules/ecmascript-6.js

```diff
@@ -19,8 +19,8 @@ module.exports = {
   'no-const-assign': 'error',
   // Disallow duplicate name in class members
   'no-dupe-class-members': 'error',
-  // Disallow duplicate module imports
-  'no-duplicate-imports': 'error',
+  // Disallow duplicate module imports (disabled, as import/no-duplicates does the same job but better)
+  'no-duplicate-imports': 'off',
```

Comment:
❤️ I just ran into this and figured I was going to have to submit a PR for that rule. Awesome that we already had a better rule in place!